packages/algob/src/builtin-tasks/run.ts

```diff
@@ -91,10 +101,12 @@ export async function runMultipleScripts (
   onSuccessFn: (cpData: CheckpointRepo, relativeScriptPath: string) => void,
   force: boolean,
   logTag: string,
-  allowWrite: boolean): Promise<void> {
+  allowWrite: boolean,
+  createAlgoClientFn: (network: Network) => AlgoDeployClient
+): Promise<void> {
```

Comment:
in all this tasks we can pass the right deployer directly, which in my opinion is better than passing all the way down a constructor / fabric method.

Reply:
I duplicated the call to `env.network` and did it.

Reply:
This is outdated code. 
Current merged master looks like this:
```
export async function runMultipleScripts (
  runtimeEnv: AlgobRuntimeEnv,
  scriptNames: string[],
  onSuccessFn: (cpData: CheckpointRepo, relativeScriptPath: string) => void,
  force: boolean,
  logTag: string,
  allowWrite: boolean,
  algoClient: AlgoDeployClient
): Promise<void> {
```